datafusion/datasource/src/statistics.rs

```diff
@@ -516,7 +516,7 @@ pub fn compute_file_group_statistics(
 /// * The summary statistics across all file groups, aka all files summary statistics
 pub fn compute_all_files_statistics(
     file_groups: Vec<FileGroup>,
-    file_schema: SchemaRef,
+    table_schema: SchemaRef,
```

Comment:
https://github.com/apache/datafusion/blob/main/datafusion/core/src/datasource/listing/table.rs#L1132 

The `self.schema()` is `table_schema`